src/types/props.ts

```diff
@@ -0,0 +1,63 @@
+import { Card, Category, Lobby, StorageKey, User } from '@/types'
+import { Ref } from 'vue'
+
+export interface CategoryCardInterface {
```

Comment:
Tutaj nie używałbym suffix'a Interface, wydaje mi się że guidline TSa tak sugeruje + sam typ często jest "self explanatory" - gdy użyjemy typ, którego strukture jest obiektem to raczej wiemy, że to interface + w mógłbym wystąpić case chęci zmiany interface na type + to zawsze dłuższa nazwa ale jestem otwarty na dyskusje tutaj:D

Reply:
No w sumie bez tego wydzielania do osobnego pliku używaliśmy bez tego Interface dodatkowo xD To faktycznie pozmieniam bo strasznie się te nazwy wydłużają